### PR TITLE
Expand reviewers/approvers and add labels for conformance image

### PR DESCRIPTION
**Pull request by dims**
We need all the help we can get to switch over sonobuoy to use the conformance image

Change-Id: I6e13d262a8bf97ada89227a853debe58876c212a

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dims** [comment]
@neolit123 @bentheelder @brahmaroutu @stevesloka @johnSchnake - folks, additional responsibility ok with you? :)

**Review by BenTheElder — APPROVED** [review]
/lgtm
/hold
(Holding for everyone else)

**Comment by neolit123** [comment]
> kubernetes-e2e-gce-100-performance — Job failed. :trollface: 

/retest

File: cluster/images/conformance/OWNERS [hunk 1]
```diff
@@ -5,10 +5,18 @@ reviewers:
   - dims
   - ixdy
   - spiffxp
+  - neolit123
+  - bentheelder
+  - brahmaroutu
+  - stevesloka
+  - johnSchnake
```
Comment (nikhita) [inline cluster/images/conformance/OWNERS]: @johnSchnake is not a @kubernetes member. John, could you [apply](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) to be a member? This should give you powers to lgtm. :)

Comment (johnSchnake) [reply cluster/images/conformance/OWNERS]: https://github.com/kubernetes/org/issues/557

**Pull request merged**